src/test/scala/github/gphat/censorinus/SynchronySpec.scala

```diff
@@ -7,16 +7,18 @@ import github.gphat.censorinus.statsd.Encoder
 class SynchronySpec extends FlatSpec with Matchers with Eventually {
 
   "Client" should "deal with gauges" in {
-    val s = new TestSender()
+    val s = new TestSender(1)
     val client = new Client(encoder = Encoder, sender = s)
 
+    // Queue up a message in the sender to ensure we can't publish yet.
+    s.buffer.offer("BOO!")
+    s.buffer.size should be (1)
```

Comment:
This is just to force our sender's `send` method to block when we attempt to send the metric.